@xen-orchestra/fs/src/abstract.js

```diff
@@ -73,6 +71,8 @@ class PrefixWrapper {
   }
 }
 
+let sharedLimit = limit(DEFAULT_MAXIMUM_PARALLEL_OPERATIONS)
```

Comment:
No, don't use global variables.

Reply:
function wrapper syntax doen't work inside class, I had to use `globalVariables`